nemo/core/classes/modelPT.py

```diff
@@ -42,6 +42,10 @@
 __all__ = ['ModelPT']
 
 
+# multiple interpolated values in the config
+OmegaConf.register_new_resolver("multiply", lambda x, y: x * y, replace=True)
```

Comment:
@ericharper heads up. ASR kinda needs the ability for Hydra multiply as well, so I'm making it a mandatory resolver by adding it into the ModelPT class (since all nemo models import it, omegaconf in NeMo usage will always support multiply op).

Keeping the support inside of hydra_runner as well - just making it so it wont crash due to already registered component.